src/data/real/nnreal.lean

```diff
@@ -131,12 +129,11 @@ instance : algebra ℝ≥0 ℝ := to_real_hom.to_algebra
 @[simp] lemma coe_to_real_hom : ⇑to_real_hom = coe := rfl
 
 instance : comm_group_with_zero ℝ≥0 :=
-{ exists_pair_ne := ⟨0, 1, assume h, zero_ne_one $ nnreal.eq_iff.2 h⟩,
-  inv_zero       := nnreal.eq $ show (0⁻¹ : ℝ) = 0, from inv_zero,
-  mul_inv_cancel := assume x h, nnreal.eq $ mul_inv_cancel $ ne_iff.2 h,
-  .. (by apply_instance : has_inv ℝ≥0),
-  .. (_ : comm_semiring ℝ≥0),
-  .. (_ : semiring ℝ≥0) }
+{ zero := 0,
+  mul := (*),
+  one := 1,
+  inv := has_inv.inv,
+  .. nnreal.coe_injective.comm_group_with_zero _ rfl rfl (λ _ _, rfl) (λ _, rfl) }
```

Comment:
```suggestion
  .. nnreal.coe_injective.comm_group_with_zero_div _ rfl rfl (λ _ _, rfl) (λ _, rfl) (λ _ _, rfl) }
```
to get a defeq division.

Reply:
We have no other `has_div` on `nnreal`.

Reply:
I think we should probably add one, but that's out of scope for this PR